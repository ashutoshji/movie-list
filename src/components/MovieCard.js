import React from 'react';

const MovieCard =({title,overview,release_date,poster_path}) => {
  const path = `https://image.tmdb.org/t/p/w185_and_h278_bestv2${poster_path}`;
  var bgStyle = {
            backgroundImage: 'url(' + path + ')'
        }

				return(
				<div className="card">
					<div className="front" style={bgStyle} title={title}>
					</div>
						<div className="back">
						 <div className="title">
						<h3 >{title}</h3>
						</div>
						<p>{overview}</p>
						<div className="date">
							Published: {release_date}
						</div>
						</div>
					
				</div>)

}

export default MovieCard;

