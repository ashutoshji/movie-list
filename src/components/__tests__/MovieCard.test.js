
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from '../MovieCard';


describe('MovieCard test', () => {
  it('MovieCard should match snapshot', () => {
    const component = renderer.create(<MovieCard
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
