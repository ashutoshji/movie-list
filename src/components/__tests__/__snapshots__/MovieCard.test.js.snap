// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard test MovieCard should match snapshot 1`] = `
<div
  className="card"
>
  <div
    className="front"
    style={
      Object {
        "backgroundImage": "url(https://image.tmdb.org/t/p/w185_and_h278_bestv2undefined)",
      }
    }
  />
  <div
    className="back"
  >
    <div
      className="title"
    >
      <h3 />
    </div>
    <p />
    <div
      className="date"
    >
      Published: 
    </div>
  </div>
</div>
`;
